pillar/heroku/mitopen.sls

```diff
@@ -168,17 +167,17 @@ heroku:
     PGBOUNCER_MAX_CLIENT_CONN: {{ env_data.PGBOUNCER_MAX_CLIENT_CONN }}
     PGBOUNCER_MIN_POOL_SIZE: {{ env_data.PGBOUNCER_MIN_POOL_SIZE }}
     PROLEARN_CATALOG_API_URL: https://prolearn.mit.edu/graphql
-    SECRET_KEY: __vault__:gen_if_missing:secret-{{ business_unit }}/django-secret-key>data>value
+    SECRET_KEY: __vault__:gen_if_missing:secret-mitopen/secrets>data>data>django-secret-key
```

Comment:
This won't work properly with nested values and `gen_if_missing`. Might be good to keep this as a separate key in the mount to avoid things getting accidentally clobbered.

Reply:
Can I just get rid of `gen_if_missing` instead? It appears to just be an arbitrary alpha-numeric string.

Reply:
Yes, that is an option. The behavior of gen_if_missing was to simplify the work of populating secrets when we didn't actually care about the value.

Reply:
I like that better. It is a one-time thing and I have a little function in my shell to generate random noise like this. I will replace the values for these three rather than carry them over from the existing mounts.

Reply:
There's also always the option of generating secrets with `python -c "import secrets; print(secrets.token_urlsafe())"`